tlslite/tlsrecordlayer.py

```diff
@@ -722,6 +722,19 @@ def _getMsg(self, expectedType, secondaryType=None, constructorType=None):
                                 yield result
                             continue
 
+                    # If we received a hearbeat request ...
+                    if recordHeader.type == ContentType.heartbeat:
```

Comment:
how do we know that the heartbeat protocol is supported by the server and negotiated for the connection?